qiita_pet/templates/study_description_templates/raw_data_info.html

```diff
@@ -0,0 +1,56 @@
+<div>
+  <div class="panel-group" id="accordion-rd-info-{{rd_id}}-{{prep_template_id}}" role="tablist" aria-multiselectable="true">
+    <div class="panel panel-default">
+      <div class="panel-heading" role="tab" id="rd-info-{{rd_id}}-{{prep_template_id}}">
+        <h4 class="panel-title">
+          <a class="collapsed" data-toggle="collapse" data-parent="#accordion-rd-info-{{rd_id}}-{{prep_template_id}}" href="#collapse-rd-info-{{rd_id}}-{{prep_template_id}}" aria-expanded="false" aria-controls="collapse-rd-info-{{rd_id}}-{{prep_template_id}}">
+            Raw data - {{rd_filetype}} (ID: {{rd_id}})
```

Comment:
As you said, it doesn't need to be an accordion. However it is convenient for multiple reasons:
 - Conceptual division of the data. This accordion makes clear that such piece of information belongs to the RawData, not necessarily to the prep tempalte
 - The user can collapse it if he doesn't want to see the RawData information
 - Last but not least, if a RawData consists of multiple files, the 'preprocess' button and any other information below the raw data can end up on the bottom of the page, so the user will need to scroll a lot to get into them (imagine a RawData in which the user has the per sample fastq...)

I think it doesn't harm anything and we get a lot of advantages of having it as an accordion, so I prefer to leave it as it is.